src/docs/product/alerts/alert-types.mdx

```diff
@@ -79,7 +87,7 @@ Crash rate alerts tell you when the crash free percentage for either [sessions o
 
 <Note>
 
-Metric alerts aren’t affected by [ignored issues](/product/issues/states-triage/#ignore), so events from those issues are counted if they match the filters of your metric alert rule.
+Metric-backed data is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in participating, [join the waitlist](https://sentry.io/for/performance/#updates-signup).
```

Comment:
I'm not sure we should remove the existing note, this still hasn't changed and affects other types of alerts

Reply:
I just moved it [further up the page](https://github.com/getsentry/sentry-docs/pull/5335/files#diff-7440c5d2e257451d1a872203cfdb4a1fa3aa933b6e5c8bfda5df2798e724915dR47) to avoid having this note right next to the beta note. I thought that would be confusing.

Reply:
Not sure what's up with my reading comprehension, thanks!